tribe-ext-pdf-tickets.php

```diff
@@ -1254,10 +1322,7 @@ public function ticket_link( $attendee_id ) {
 				return '';
 			}
 
-			$output = sprintf(
-				'<a href="%s"',
-				$url
-			);
+			$output = sprintf( '<a href="%s"', $url);
```

Comment:
This should have esc_url() in it and you should remove the esc_url() being called above.

Reply:
`esc_url()` happened upstream to ensure the text is actually a URL - and if not then don't output the HTML for a link

Reply:
You can set the `$pdf_link = $this->get_pdf_link( $attendee_id );` then check `if ( empty( esc_url_raw( $url ) )` and then output this as 
```suggestion
			$output = sprintf( '<a href="%s"', esc_url( $url ) );
```